src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs

```diff
@@ -242,13 +242,13 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 Document document,
                 SyntaxEditor editor,
                 SyntaxNode root,
-                IEnumerable<SyntaxNode> declarationsToFix,
+                ImmutableArray<SyntaxNode> declarationsToFix,
                 UseExpressionBodyHelper helper,
                 bool useExpressionBody,
                 CancellationToken cancellationToken)
             {
                 // Process all declaration nodes in reverse to handle nested declaration updates properly.
-                declarationsToFix = declarationsToFix.Reverse();
+                declarationsToFix = declarationsToFix.Reverse().ToImmutableArray();
```

Comment:
sorry to be annoying.  but i would just remove this line entirely, and then change the foreach to foreach over the .Reverse.  taht shoudl be safe since all the nodes are realized in 'declarationsToFix'.